src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnectionX.cs

```diff
@@ -85,13 +153,50 @@ public override Task ChangeDatabaseAsync(string databaseName, CancellationToken
         public object Clone()
             => throw new NotImplementedException();
 
+        #region close
+
         /// <inheritdoc/>
         public override void Close()
-            => throw new NotImplementedException();
+            => Close(async: false).GetAwaiter().GetResult();
 
         /// <inheritdoc/>
         public override Task CloseAsync()
-            => throw new NotImplementedException();
+            => Close(async: true);
```

Comment:
ConfigureAwait(false)

Reply:
I can only add this on lines that directly await a Task with the await keyword. I fixed one other spot where it was missing though.